RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -899,420 +955,400 @@ class DeepTauId : public deep_tau::DeepTauBase {
+                getValueNorm(electrons.at(index_ele).deltaEtaSuperClusterTrackAtVtx(),0.f, 0.0363f);
+            get(dnn::ele_deltaEtaSeedClusterTrackAtCalo) =
+                getValueNorm(electrons.at(index_ele).deltaEtaSeedClusterTrackAtCalo(), -0.0001f, 0.0512f);
+            get(dnn::ele_deltaEtaEleClusterTrackAtCalo) =
+                getValueNorm(electrons.at(index_ele).deltaEtaEleClusterTrackAtCalo(), -0.0001f, 0.0541f);
+            get(dnn::ele_deltaPhiEleClusterTrackAtCalo) =
+                getValueNorm(electrons.at(index_ele).deltaPhiEleClusterTrackAtCalo(), 0.0002f, 0.0553f);
+            get(dnn::ele_deltaPhiSuperClusterTrackAtVtx) =
+                getValueNorm(electrons.at(index_ele).deltaPhiSuperClusterTrackAtVtx(), 0.0001f, 0.0523f);
+            get(dnn::ele_deltaPhiSeedClusterTrackAtCalo) =
+                getValueNorm(electrons.at(index_ele).deltaPhiSeedClusterTrackAtCalo(), 0.0004f, 0.0777f);
+            get(dnn::ele_mvaInput_earlyBrem) = getValue(electrons.at(index_ele).mvaInput().earlyBrem);
+            get(dnn::ele_mvaInput_lateBrem) = getValue(electrons.at(index_ele).mvaInput().lateBrem);
+            get(dnn::ele_mvaInput_sigmaEtaEta) = getValueNorm(electrons.at(index_ele).mvaInput().sigmaEtaEta,0.0008f, 0.0052f);
+            get(dnn::ele_mvaInput_hadEnergy) = getValueNorm(electrons.at(index_ele).mvaInput().hadEnergy, 14.04f, 69.48f);
+            get(dnn::ele_mvaInput_deltaEta) = getValueNorm(electrons.at(index_ele).mvaInput().deltaEta, 0.0099f, 0.0851f);
+
+            const auto& gsfTrack = electrons.at(index_ele).gsfTrack();
+            if(gsfTrack.isNonnull()){
+                get(dnn::ele_gsfTrack_normalizedChi2) = getValueNorm(gsfTrack->normalizedChi2(), 3.049f, 10.39f);
+                get(dnn::ele_gsfTrack_numberOfValidHits) = getValueNorm(gsfTrack->numberOfValidHits(), 16.52f, 2.806f);
+                get(dnn::ele_rel_gsfTrack_pt) = getValueNorm(gsfTrack->pt() / electrons.at(index_ele).polarP4().pt(), 1.355f, 16.81f);
+                get(dnn::ele_gsfTrack_pt_sig) = getValueNorm(gsfTrack->pt() / gsfTrack->ptError(), 5.046f, 3.119f);
+            }
+            const auto& closestCtfTrack = electrons.at(index_ele).closestCtfTrackRef();
+            const bool has_closestCtfTrack = closestCtfTrack.isNonnull();
+            if(has_closestCtfTrack){
+                get(dnn::ele_has_closestCtfTrack) = has_closestCtfTrack;
+                get(dnn::ele_closestCtfTrack_normalizedChi2) = getValueNorm(closestCtfTrack->normalizedChi2(), 2.411f, 6.98f);
+                get(dnn::ele_closestCtfTrack_numberOfValidHits) = getValueNorm(closestCtfTrack->numberOfValidHits(), 15.16f, 5.26f);
             }
         }
-        checkInputs(inputs, is_inner ? "egamma_inner_block" : "egamma_outer_block", dnn::NumberOfInputs, grid.nCellsEta,
-            grid.nCellsPhi);
+        // checkInputs(inputs, is_inner ? "egamma_inner_block" : "egamma_outer_block", dnn::NumberOfInputs, grid.nCellsEta,
```

Comment:
Is there any important reason to keep this code commented out? If, so please comment on this in code otherwise remove.